static/app/views/projectInstall/createProject.tsx

```diff
@@ -423,6 +423,37 @@ export function CreateProject() {
         return;
       }
 
+      if (
+        value.type === 'console' &&
+        !organization.enabledConsolePlatforms?.includes(value.id)
+      ) {
+        updateFormData('platform', {
+          category: formData.platform?.category,
+        });
+        const {ConsoleModal, modalCss} = await import(
+          'sentry/components/onboarding/consoleModal'
+        );
```

Comment:
Out of curiosity: why are we doing this in an on-demand-import?

Reply:
Yes, this is a pretty standard pattern we use right now. I just copied and pasted it from other places where we’ve already done the same thing.